src/json.cc

```diff
@@ -224,7 +226,35 @@ Obj JSON_ESCAPE_STRING(Obj self, Obj param)
     return copy;
 }
 
+Obj GAP_PLIST_TO_JSON_STRING(Obj self, Obj string, Obj stream, Obj list) {
```

Comment:
```suggestion
Obj GAP_PLIST_TO_JSON_STRING(Obj self, Obj string, Obj stream, Obj list)
{
```
For consistency with the rest of the code in this file?